tests/nightly/test_large_array.py

```diff
@@ -279,6 +279,33 @@ def test_diag():
     assert_almost_equal(r.asnumpy(), np.diag(a_np, k=k))
 
 
+def create_2d_tensor(rows, columns):
+    a = np.arange(0, rows).reshape(rows, 1)
+    b = np.broadcast_to(a, shape=(a.shape[0], columns))
+    return nd.array(b, dtype=np.int64)
```

Comment:
can we just return b?

Reply:
we need ndarray so we need to convert it

Reply:
w/o int64 conversion the data somehow gets truncated at certain value do dtype is also required

Reply:
adding conversion to same line makes it too long. I felt to keep that in next line makes it more readable. Also, do you think adding return statement makes it less pythonic ?